src/Support/Lazy/InertiaDeferred.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Spatie\LaravelData\Support\Lazy;
+
+use Inertia\DeferProp;
+
+class InertiaDeferred extends ConditionalLazy
+{
+    public function __construct(
+        mixed $value,
```

Comment:
This is `mixed` instead of `DeferProps', since the `$value` parameter in the `AutoInertiaDeferred::build` method is mixed and needs to be passed to this function